cmd/operator-sdk/generate/crds.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package generate
+
+import (
+	"fmt"
+
+	"github.com/operator-framework/operator-sdk/cmd/operator-sdk/internal/genutil"
+	"github.com/spf13/cobra"
+)
+
+func newGenerateCRDsCmd() *cobra.Command {
```

Comment:
can we add a flag for passing in the location of the CRD's?

Reply:
Once this merges, I'll make an issue to discuss this idea for a follow-up.